docs/requirements/stpod/011-swift-layer2-sync.robot

```diff
@@ -0,0 +1,15 @@
+*** Settings ***
+Documentation     Swift VMs need connectivity to each other for sync purposes.
+...
```

Comment:
Could we maybe mention which of the previous networks is supposed to be used for this? I assume `Controlplane sync network`